packages/babel-plugin-minify-simplify/src/index.js

```diff
@@ -124,6 +135,49 @@ module.exports = ({ types: t }) => {
         ],
       },
 
+      AssignmentExpression(path) {
+
+        const right = path.get('right');
+        const left = path.get('left');
+
+        const canShorten = (
+          right.type === 'BinaryExpression' &&
+          operators.has(right.node.operator) &&
+          left.node.name === right.node.left.name
+        );
+
+		    if (!canShorten) return;
+
+        const canBeUpdateExpression = (
+          right.node.right.type === 'NumericLiteral' &&
+          right.node.right.value === 1 &&
+          updateOperators.has(right.node.operator));
+
+        if (left.node.name === undefined) {
+          // TODO: transform MemberExpressions as well
+          // e.g. foo.bar = foo.bar + 123;
+          return;
+        }
+
+        if (canBeUpdateExpression) {
+
+          const newExpression = t.updateExpression(
+            right.node.operator + right.node.operator,
+            t.identifier(left.node.name));
```

Comment:
Wonder if we can make a generic utility for handling both Identifier/MemberExpression?